File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Selvstendig.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

import java.util.List;


@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
@Data
public class Selvstendig {

    private List<Arbeidsgiver> arbeidsgiver;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Selvstendig.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Selvstendig (
    var arbeidsgiver: List<Arbeidsgiver>? = null
)
